### ISSUE TITLE
Do you meet this problem when you train CLIP from scratch?

### ISSUE THREAD

**Issue by qiaogh97** [revantteotia/clip-training#4]
Hello, @revantteotia !
I use this code to train CLIP from scratch using CC3M + CC12M dataset. But my loss curve seems to be weird as below.
![image](https://user-images.githubusercontent.com/20733440/147475791-328bf1ff-02d4-4393-97df-824e5510e3d8.png)
As you can see, the loss would rise suddenly and drop faster than before. Do you meet the same problem?
PS: 1 epoch = 6700 step

**Comment by revantteotia** [comment]
Hi @qiao1025566574 , I did not train it using CC3M + CC12M dataset. I just trained it on a smaller dataset (COCO captions) to see if the training code is working properly.  For COCO captions, I did not see such issues.

**Comment by qiaogh97** [comment]
Thank you for your reply!
